app/models/donation.rb

```diff
@@ -0,0 +1,19 @@
+class Donation < ApplicationRecord
+  belongs_to :girl
+  serialize :notification_params, Hash
+
+  def paypal_url(return_path)
+    values = {
+      business: "admin@insaka.org",
+      cmd: "_xclick",
+      upload: 1,
+      return: "#{Rails.application.secrets.app_host}#{return_path}",
+      invoice: id,
+      amount: amount,
+      description: "test foo thingggy",
```

Comment:
need to figure out which fields the paypal api'll accept cause this one isn't showing up